src/docker-entrypoint.sh

```diff
@@ -48,7 +48,7 @@ function generate_users() {
     else
       echo -e "$username\t<set by secrets file>"
     fi
-    adduser "$username" --quiet --disabled-password --shell /usr/sbin/nologin --gecos "" &> /dev/null || true
+    adduser "$username" --quiet --disabled-password --shell /usr/sbin/nologin --gecos "" --force-badname &> /dev/null || true
```

Comment:
@dav3r, does it make sense do drop the `&> /dev/null` here, and elsewhere in the file if this isn't the only occurrence?  The hiding of error messages is something that I thought was problematic about the old ServerSetup script as well.

Reply:
Can you think of any security reasons why we shouldn't include this output?  Nothing is coming to mind at this moment, but I want to put it out there in case someone has a good idea.

Otherwise, I'd be fine with dropping the `&> /dev/null` here and elsewhere.

Reply:
See [here](https://github.com/cisagov/postfix-docker/pull/33#pullrequestreview-853078541).